Silicon/ApollolakePkg/Library/FirmwareUpdateLib/FirmwareUpdateLib.c

```diff
@@ -286,6 +286,46 @@ GetFirmwareUpdateInfo (
   return EFI_SUCCESS;
 }
 
+/**
+  Retrieve the current in-flight FW update state, if it exists
+
+  @param[out] State The current in-flight FW update state
+**/
+VOID
+EFIAPI
+GetInFlightUpdateState (
```

Comment:
This is defined separately to APL. 
Any specific reason?

Reply:
When you include FirmwareResiliencyLib in this file, SetBootPartition conflicts between FirmwareUpdateLib and TopSwapLib for APL. To avoid this, I just copied over the needed API.